src/redux/actions/authUser.js

```diff
@@ -0,0 +1,33 @@
+// Signup user
```

Comment:
Remove comment. It's irrelevant at this point.

Reply:
on it